python/paddle/autograd/autograd.py

```diff
@@ -449,8 +458,8 @@ def _multi_index(indexes, shape):
 
 
 def jacobian(
-    ys: paddle.Tensor | tuple[paddle.Tensor, ...],
-    xs: paddle.Tensor | tuple[paddle.Tensor, ...],
+    ys: Tensor | tuple[Tensor, ...],
+    xs: Tensor | tuple[Tensor, ...],
```

Comment:
```suggestion
    ys: Tensor | Sequence[Tensor, ...],
    xs: Tensor | Sequence[Tensor, ...],
```

另外，这里貌似需要用 `overload` ，`ys, xs` 任何一个为 `Sequence` ，返回 `tuple[Jacobian, ...]` 否则只返回一个 `Jacobian`

Reply:
收到